src/comodo/jaxsimSimulator/jaxsimSimulator.py

```diff
@@ -0,0 +1,570 @@
+        contact_params: ContactsParams | None = None,
+        left_foot_link_name: str | None = "l_ankle_2",
+        right_foot_link_name: str | None = "r_ankle_2",
+        left_foot_sole_frame_name: str | None = "l_sole",
+        right_foot_sole_frame_name: str | None = "r_sole",
+        visualization_mode: str | None = None,
+    ) -> None:
+        """
+        Load and initialize the robot model for simulation.
+
+        Args:
+            robot_model: The robot model to be loaded.
+            dt: The time step for the simulation in seconds.
+            xyz_rpy: The initial position and orientation (roll, pitch, yaw) of the robot base.
+            contact_model_type: The type of contact model to use.
+            s: The initial joint positions.
+            contact_params: The parameters for the contact model.
+            left_foot_link_name: The name of the left foot link.
+            right_foot_link_name: The name of the right foot link.
+            left_foot_sole_frame_name: The name of the left foot sole frame.
+            right_foot_sole_frame_name: The name of the right foot sole frame.
+            visualization_mode: The mode for visualization, either "record" or "interactive".
+
+        Raises:
+            ValueError: If an invalid contact model type or visualization mode is provided.
+        """
+        # ==== Initialize simulator model and data ====
+
+        self._dt = dt
+        self._t = 0.0
+        self._contact_model_type = contact_model_type
+
+        match contact_model_type:
+            case JaxsimContactModelEnum.RIGID:
+                contact_model = RigidContacts.build()
+            case JaxsimContactModelEnum.RELAXED_RIGID:
+                contact_model = RelaxedRigidContacts.build()
+            case JaxsimContactModelEnum.VISCO_ELASTIC:
+                contact_model = ViscoElasticContacts.build()
+            case JaxsimContactModelEnum.SOFT:
+                contact_model = SoftContacts.build()
+            case _:
+                raise ValueError(f"Invalid contact model type: {contact_model_type}")
+
+        model = js.model.JaxSimModel.build_from_model_description(
+            model_description=robot_model.urdf_string,
+            model_name=robot_model.robot_name,
+            contact_model=contact_model,
+        )
+
+        self._model = js.model.reduce(
+            model=model,
+            considered_joints=robot_model.joint_name_list,
+        )
+
+        if contact_params is None:
+            match contact_model_type:
+                case JaxsimContactModelEnum.RIGID:
+                    contact_params = RigidContactsParams.build(mu=0.5, K=1.0e4, D=1.0e2)
+                case JaxsimContactModelEnum.RELAXED_RIGID:
+                    contact_params = RelaxedRigidContactsParams.build(mu=0.005)
+                case JaxsimContactModelEnum.VISCO_ELASTIC | JaxsimContactModelEnum.SOFT:
+                    contact_params = js.contact.estimate_good_soft_contacts_parameters(
```

Comment:
As in https://github.com/ami-iit/jaxsim/pull/268
```suggestion
                    contact_params = js.contact.estimate_good_contact_parameters(
```

Reply:
Thanks!